### ISSUE TITLE
How do I use it with Angular4?

### ISSUE THREAD

**Issue by NicCyclePath** [s-yadav/angulargrid#138]
It may sound like a dumb question but I just start learning angular, and This would help me a lot with my project. Thanks you guys in advance :)

**Comment by musicsmithnz** [comment]
I agree there is no installation guide the linked documentation is here:

http://ignitersworld.com/lab/angulargrid/index.html#documentation

But this documentation doesn't show installation. It shows particulars of what to do inside particular files. The Documentation needs work. You shouldn't need to understand the whole project before using it. Modular code means that you don't need to understand code before you use it.
